### PR TITLE
Added Currency module

### PR DESCRIPTION
**Pull request by Nithesh1501**
Please review and merge it to your code. Glad to add another module to this project.
Thanks @urstrulycherry

### CODE REVIEW

File: modules/currency.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+const process = async (message: WAWebJS.Message, _client: WAWebJS.Client) => {
+    console.log("economy");
+    const msg = message.body.split(" ").slice(1).join(" ");
+    const arr = msg.split(" ").filter((item) => item.trim());
+    const [cur1, cur2] = arr;
+    if (!cur1) {
+        send.text(message, "Please provide currency code(s)");
+        return;
+    }
+    let res;
```
Comment (urstrulycherry) [inline modules/currency.ts]: Use ternary operator instead of if-else and make "let res" as "const res"

File: modules/currency.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+    if (!res) {
+        send.text(message, incorrect);
+        return;
+    }
+    send.text(message, res);
+};
+
+const trigger = async (cur1: string, cur2: string | undefined) => {
+    let url;
```
Comment (urstrulycherry) [inline modules/currency.ts]: Same here

File: modules/currency.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    const res = await axios.get(url).catch(() => {
+        return null;
+    });
+
+    if (!res) return;
+
+    if (res.status !== 200) {
+        return null;
+    }
+    let msg;
```
Comment (urstrulycherry) [inline modules/currency.ts]: Here too..!

**Review by urstrulycherry — CHANGES_REQUESTED** [review]

File: modules/currency.ts (hunk 3 above)
Comment (Nithesh1501) [reply modules/currency.ts]: Done.

File: modules/currency.ts (hunk 2 above)
Comment (Nithesh1501) [reply modules/currency.ts]: Changes were made.

File: modules/currency.ts (hunk 1 above)
Comment (Nithesh1501) [reply modules/currency.ts]: Code Fixed.

**Review by urstrulycherry — APPROVED** [review]
Great...!

**Pull request merged**